ext/reflection/tests/ReflectionType_001.phpt

```diff
@@ -2,7 +2,7 @@
 ReflectionParameter::get/hasType and ReflectionType tests
 --FILE--
 <?php
-function foo(stdClass $a, array $b, callable $c, stdClass $d = null, $e = null, string $f, bool $g, int $h, float $i, NotExisting $j) { }
+function foo(stdClass $a, array $b, ?callable $c, stdClass $d = null, $e = null, string $f, bool $g, int $h, float $i, NotExisting $j) { }
```

Comment:
This new case should be handled in a separate test, to be able to skip it on PHP < 7.1